plugins/chunter-resources/src/components/CommentInput.svelte

```diff
@@ -17,14 +17,105 @@
   import { Comment } from '@hcengineering/chunter'
   import { Doc, generateId, Ref } from '@hcengineering/core'
   import { getClient } from '@hcengineering/presentation'
+  import { fetchMetadataLocalStorage, setMetadataLocalStorage } from '@hcengineering/ui'
   import { AttachmentRefInput } from '@hcengineering/attachment-resources'
+  import { writable } from 'svelte/store'
   import { createBacklinks } from '../backlinks'
   import chunter from '../plugin'
 
-  const client = getClient()
   export let object: Doc
+  export let shouldUseDraft: boolean = false
+
+  const client = getClient()
   const _class = chunter.class.Comment
+  const drafts = writable<Record<string, Comment>>()
```

Comment:
Better to have a writable with <Id, DraftDoc> and DraftDoc should contain any of document fields and attachments as well, as any other serialisable fields.

So it could be easy reused lately if more document drafts will be required, and chaning only this part will be possible to move all drafts on server DB>